services/reflex/knowledge_base_construction/src/run_experiments.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+import time
+import pandas as pd
+import argparse
+from roberta_completion import main as base_evaluation
+import pprint
+import statistics
+from collections import defaultdict
+from roberta_connector import Roberta
+from context_filtering.context_filtering import Context_Filtering
+import sys
+base_model = True
+use_filter = False
+zsre_analysis = False
+MODEL_NAME = 'model.pt'
+we_model_pth = '/data/cc.en.300.bin' 
+db_dump = '/data/contexts.db'
+word_weight_pth = '/data/wiki_word_weight.json'
+num_random_contexts = 1000
```

Comment:
Whatever is necessary for the context filtering, move into the Init function for Context_Filtering

The things that look like configs, pass as __init__ parameters